curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -243,8 +292,29 @@ int AddWarmupTask(curvefs::client::common::WarmupType type, fuse_ino_t key,
     return ret;
 }
 
+int CancelWarmupTask(curvefs::client::common::WarmupType type, fuse_ino_t key) {
+    int ret = 0;
+    bool result = true;
+    switch (type) {
+    case curvefs::client::common::WarmupType::kWarmupTypeList:
+        result = g_ClientInstance->RemoveWarmFilelistTask(key);
+        break;
+    case curvefs::client::common::WarmupType::kWarmupTypeSingle:
+        result = g_ClientInstance->RemoveWarmFileTask(key);
+        break;
```

Comment:
Why do you want to separate the two? It can be indistinguishable.

Reply:
combined